src/components/common/YourCompany/YourCompany.vue

```diff
@@ -212,6 +212,16 @@
 
     <v-divider class="mx-4 my-1" />
 
+    <!--- Association Type (coop only) -->
+    <div v-if="isEntityTypeCP"
+        id="association-type-section"
+        class="section-container">
```

Comment:
Our code style is to put the closing bracket on a new line, in line with the start bracket, when this is a multi-line element/component. Eg, see line 206.